app/pages/weekly/[year].tsx

```diff
@@ -0,0 +1,103 @@
+import { GetServerSideProps } from 'next';
+import Head from 'next/head';
+import Link from 'next/link';
+import { DefaultTopBar } from '../../components/TopBar.js';
+import { ArticleT } from '../../lib/article.js';
+import { getWeeklyEmailsFromYear } from '../../lib/serverOnly';
+
+interface WeeklyEmailProps {
+  articles: ArticleT[];
+  year: string;
+}
+
+export const getServerSideProps: GetServerSideProps<WeeklyEmailProps> = async ({
```

Comment:
I think the only change we need to make here is to add the `withTranslation` wrapper around `getServerSideProps` - that adds in the props we need to do i18n. Take a look at pages/index.tsx for an example - basically rename this function to something else and then export getServerSideProps as that new function wrapped with `withTranslation`.